.github/workflows/build_docs.yaml

```diff
@@ -0,0 +1,34 @@
+name: Build Documentation
+
+on:
+  push:
+    paths:
+      - 'jobbergate-docs/docs/**'
```

Comment:
This may not update the docs with new docstrings in case they were updated on the API, CLI...

Reply:
Good point!

Reply:
Addressed in commit d68a6dc.